action.yml

```diff
@@ -62,16 +62,39 @@ runs:
           $response = Invoke-WebRequest -UseBasicParsing -Headers $headers -Uri "${{ inputs.pull_request_url }}" | ConvertFrom-Json
           $backportPRNumber = $response.number
 
-          $parameters = @{
-            BackportRepoName = "$repoName";
-            BackportRepoOrg = "$repoOwner";
-            BackportTargetBranch = "$backportTargetBranch";
-            BackportPRNumber = "$backportPRNumber";
-            BackportHeadSHA = "$($response.head.sha)";
-            UseFork = "${{ inputs.use_fork }}" -eq "true";
-          } | ConvertTo-Json -Compress
-
-          $json = $parameters.Replace("`"","'")
+          # we can have a comment that wants to backport to several PRs, that would mean that we create a backport per branch 
+
+          $parameters = @()
+
+          if ($backportTargetBranch.GetType().Name -eq "String") { # uh, this is ugly, but String also has length and we can iterate, so we need the type
```

Comment:
Can't we use `$backportTargetBranch -is [System.String]` instead?

Right now, `MyLib.String` also matches.

Reply:
I'm not a pwsh person ;) looks better, let me test

Reply:
Updated, works like a charm.